backend/api/schemas/copy.go

```
@@ -0,0 +1,26 @@
+	repositoryschemas "polimane/backend/repository/schemas"
+)
+
+func apiCopy(ctx *fiber.Ctx) error {
+	schemaId, err := base.GetRequiredParam(ctx, "schemaId")
+	if err != nil {
+		return err
+	}
+
+	user := auth.GetSessionUser(ctx)
+
+	schema, err := repositoryschemas.Copy(ctx.Context(), &repositoryschemas.CopyOptions{
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)